### PR TITLE
GooglePlaces-2.4.0

### PR DESCRIPTION
**Pull request by suggyd**
I've updated the bindings project and cake build to work with Google Places 2.4.0 and migrated the sample project to AndroidX and Android 11.  

Places 2.4.0 no longer organises the obfuscated classes neatly under .internals namespaces but it does appear to prefix all obfuscated classed with 'zz' so I've used a rule based on that instead.

I wasn't sure whether to take latest versions of the dependencies or the minimum version required by the Google Places 2.4.0 library?

I had a problem with Xamarin.Google.Android.DataTransport.TransportBackendCct and had to include Xamarin.Google.Dagger@2.25.2.1 to resolve it.

First time pull request and bindings library so you might need some extra patience.

### CODE REVIEW

**Comment by moljac** [comment]
@suggyd 
Thanks for your effort

> migrated the sample project to AndroidX and Android 11.

Excellent.

> Places 2.4.0 no longer organises the obfuscated classes neatly under .internals namespaces but it does appear to prefix all obfuscated classed with 'zz' so I've used a rule based on that instead.

`zz` prefix is result of obfuscation. Using rule is OK.

> I wasn't sure whether to take latest versions of the dependencies or the minimum version required by the Google Places 2.4.0 library?

I usually go with minimum (published eariler and thus used longer)

> I had a problem with Xamarin.Google.Android.DataTransport.TransportBackendCct and had to include Xamarin.Google.Dagger@2.25.2.1 to resolve it.

Possible, but I would need some details.

> First time pull request and bindings library so you might need some extra patience.

No problems and do not worry. I will take a look into the code ASAP

File: Android/GooglePlaces/source/GooglePlaces/GooglePlaces.csproj [hunk 1]
```diff
@@ -1,7 +1,7 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <Project Sdk="MSBuild.Sdk.Extras/2.0.54">
   <PropertyGroup>
-    <TargetFrameworks>MonoAndroid90</TargetFrameworks>
+    <TargetFrameworks>MonoAndroid11.0</TargetFrameworks>
```
Comment (moljac) [inline Android/GooglePlaces/source/GooglePlaces/GooglePlaces.csproj]: Please leave here 9.0.

11.0 will force all libraries and/or apps that depend/user this library to bump targetFramework and could make a lot of users unhappy.

**Review by moljac — CHANGES_REQUESTED** [review]
Please revert targetFramework to 9.0 for the bindings

File: Android/GooglePlaces/source/GooglePlaces/GooglePlaces.csproj (hunk 1 above)
Comment (suggyd) [reply Android/GooglePlaces/source/GooglePlaces/GooglePlaces.csproj]: Thanks for feedback moljac, I also bumped the ANDROID_SDK_BUILD_TOOLS version from 28.0.3 to 30.0.2 in build.cake file, should I revert that too?

Comment (moljac) [reply Android/GooglePlaces/source/GooglePlaces/GooglePlaces.csproj]: this should be OK. (I think, not sure)

Comment (suggyd) [reply Android/GooglePlaces/source/GooglePlaces/GooglePlaces.csproj]: @moljac I've reverted to monoandroid90 but I still had a query about whether you wanted me to revert the SDK build tools version back to 28.0.3?

Comment (moljac) [reply Android/GooglePlaces/source/GooglePlaces/GooglePlaces.csproj]: no 30.0.2 should be OK. We'll see soon. OK?

**Review by moljac — APPROVED** [review]
LGTM

File: Android/GooglePlaces/source/GooglePlaces/GooglePlaces.csproj (hunk 1 above)
Comment (suggyd) [reply Android/GooglePlaces/source/GooglePlaces/GooglePlaces.csproj]: Yep, I'm good.  What happens now?

Comment (suggyd) [reply Android/GooglePlaces/source/GooglePlaces/GooglePlaces.csproj]: @moljac are you waiting for me to do something?  This is my first time so not too sure of the process.

Comment (moljac) [reply Android/GooglePlaces/source/GooglePlaces/GooglePlaces.csproj]: No. Working on huge AndroidX update. Hope to be back soon

**Review by moljac — APPROVED** [review]
LGTM

**Pull request merged**